_tools/networks.ts

```diff
@@ -132,5 +132,16 @@ export const NETWORKS = [
         source: "directory-harmony-testnet-shard-0.json",
       },
     ],
+  },
+  {
+    page: "solana-price-feeds",
+    title: "Solana Price Feeds",
+    networks: [
+      {
+        name: "Solana Devnet",
+        url: "https://explorer.solana.com/address/",
```

Comment:
```suggestion
        url: "https://explorer.solana.com/?cluster=devnet",
```
Use the Devnet specific URL.

Reply:
The problem with this is that it will not open an address in the explorer. So we have an option of either pointing to the devnet cluster without an address or to the default explorer link (mainnet) with an address.

I thought it would be weird if you click on the address and it opens a blank explorer for you so I picked the second option. Thoughts?

Reply:
I tried this as a fix, but it's not a very elegant solution. It does work. https://github.com/dwightjl/documentation/commit/9d36974d5951f4ba121f9164b94b6f38291b5f7c